drivers/platform/linux/linux_gpio.c

```diff
@@ -50,6 +50,21 @@
 #include <unistd.h>
 #include <sys/ioctl.h>
 
+/******************************************************************************/
```

Comment:
commit title should be I thing 75 characters long

Reply:
What do you mean? It has only 65 characters. "... until gpio_remove() is called." is the description :).